web/gds-user-ui/src/hooks/useUploadFile.tsx

```diff
@@ -40,7 +40,7 @@ const useUploadFile = () => {
     reader.onload = async (ev: any) => {
       const data = JSON.parse(ev.target.result);
       try {
-        const validationData = await validationSchema[0].validate(data, { abortEarly: false });
+        const validationData = await allValidationSchema.validate(data);
```

Comment:
The validation should be applied to all schemas, not just the basic form. The `allValidationSchema` encompasses all the form schemas.